app/models/task.py

```diff
@@ -1,6 +1,34 @@
 from flask import current_app
 from app import db
+from .goal import Goal
+from sqlalchemy.orm import relationship
+from sqlalchemy import ForeignKey
+
 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    __tablename__ = 'task'
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, ForeignKey('goal.goal_id'), nullable=True)
+
+
+    def to_json(self):
+
+        task_dict = {
+            "id": self.task_id,
+            "title": self.title,
+            "description": self.description,
+            "is_complete": False if self.completed_at is None else True
+        }
+
+        if self.goal_id:
+            task_dict["goal_id"] = self.goal_id
+
+        return task_dict
```

Comment:
Nice helper method here. This is a good way to adding in a `goal_id` only IF a non-null value is detected. The ternary here looks great as well. Another way you can write that is using:

```suggestion
    def to_json(self):

        task_dict = {
            "id": self.task_id,
            "title": self.title,
            "description": self.description,
            "is_complete": self.completed_at != None
        }

        if self.goal_id:
            task_dict["goal_id"] = self.goal_id

        return task_dict
```